libraries/Microsoft.Bot.Builder.Dialogs.Declarative/Resources/FolderResourceProvider.cs

```diff
@@ -11,29 +11,29 @@
 namespace Microsoft.Bot.Builder.Dialogs.Declarative.Resources
 {
     /// <summary>
-    /// Folder/FileResources.
+    /// Class which gives ResourceExplorer to resources which are stored in file system.
```

Comment:
>t [](start = 43, length = 1)

?

Reply:
access to

---
In reply to: [369913812](https://github.com/microsoft/botbuilder-dotnet/pull/3290#discussion_r369913812) [](ancestors = 369913812)